app/_src/gateway/production/networking/configure-tls.md

```diff
@@ -0,0 +1,254 @@
+        ```
+3. Generate the server private key and certificate:
+    1. Generate a server private key and certificate signing request (CSR):
+        ```sh
+        openssl req -new -utf8 -nodes -subj "/CN=dbhost.yourdomain.com" -config /etc/ssl/openssl.cnf -keyout server.key -out server.csr
+        ```
+    1. Change the permissions of the private key:
+        ```sh
+        chmod og-rwx server.key
+        ```
+    1. Create a server certificate signed by the intermediate CA:
+        ```sh
+        openssl x509 -req -in server.csr -days 365 -CA intermediate.crt -CAkey intermediate.key -CAcreateserial -out server.crt
+        ```
+4. Generate a private key and certificate for the client:
+    1. Generate a client private key and certificate signing request (CSR):
+        ```sh
+       openssl req -new -utf8 -nodes -subj "/CN=kong" -config /etc/ssl/openssl.cnf -keyout client.key -out client.csr
+       ```
+    1.  Change the permissions of the private key:
+        ```sh
+        chmod og-rwx client.key
+        ```
+    1. Create a client certificate signed by an intermediate CA: 
+        ```sh
+        openssl x509 -req -in client.csr -days 365 -CA intermediate.crt -CAkey intermediate.key -CAcreateserial -out client.crt
+        ```
+5. Append the certificates:
+    1. Complete the CA chain:
+        ```sh
+        cat intermediate.crt > chain.crt
+        cat root.crt >> chain.crt
+        ```
+    2. **Optional**: append `intermediate.crt` to `server.crt`:
+        ```sh
+        cat intermediate.crt >> server.crt
+        ```
+        This step is typically done when the server is configured to use a certificate chain that includes the intermediate CA. If the server is not configured to use a certificate chain, or if the intermediate CA is already included in the `server.crt` file, this step is not necessary. 
+    3. **Optional**: append `intermediate.crt` to `client.crt`:
+        ```sh
+        cat intermediate.crt >> client.crt
+        ```
+        This step is typically done when the client is configured to use a certificate chain that includes the intermediate CA. If the client is not configured to use a certificate chain, or if the intermediate CA is already included in the `client.crt` file, this step is not necessary. 
+
+### Chain without intermediate CA
+
+In some cases, the certificate chain can be signed directly by the root certificate authority instead of an intermediate CA. This type of chain may be used in situations where the certificate is issued by a well-known and trusted root CA, and the level of security and trust provided by an intermediary CA is not necessary. 
+
+{:.important}
+> **Important:** Not using an intermediary CA can be less secure than using an intermediary CA, because the chain is shorter, and there are no additional layers of security that would normally be provided by the intermediary. 
```

Comment:
```suggestion
> **Important:** Not using an intermediate CA can be less secure than using an intermediate CA, because the chain is shorter, and there are no additional layers of security that would normally be provided by the intermediate CA. 
```